package.json

```diff
@@ -190,6 +192,7 @@
     "url-loader": "^1.0.1",
     "vrtest": "^0.2.0",
     "webfontloader": "^1.6.28",
+    "webpack": "^4.28.4",
```

Comment:
Was a transitive dependency of `size-limit` and `rollup-plugin-size-snapshot`

Reply:
It's not true.
https://github.com/TrySound/rollup-plugin-size-snapshot/blob/master/package.json#L72

Reply:
Right, I mispoke. What I wanted to say was:
> Is a direct dependency of [...]